dawn/src/dawn/CodeGen/Cuda-ico/CudaIcoCodeGen.cpp

```diff
@@ -935,6 +935,11 @@ void CudaIcoCodeGen::generateAllAPIRunFunctions(
   }
 }
 
+std::string getEnvVar(const std::string key) {
+    char * val = std::getenv(key.c_str());
+    return val == NULL ? std::string("") : std::string(val);
+}
+
```

Comment:
Use `nullptr` instead of `NULL`. If you are interested why: https://www.geeksforgeeks.org/understanding-nullptr-c/
Instead of `std::string("")` the default constructor of `std::string` also results in an empty string. So you can use `std::string()`.

Also is this version of getEnvVar even needed? This would run at compile time, not at runtime. I think this can be deleted?